contracts/interfaces/zap/ISolidZapDecollateralize.sol

```diff
@@ -3,11 +3,76 @@ pragma solidity 0.8.18;
 
 /// @author Solid World
 interface ISolidZapDecollateralize {
+    event ZapDecollateralize(
+        address indexed receiver,
+        address indexed inputToken,
+        uint indexed inputAmount,
+        uint dust,
+        address dustReceiver,
+        uint categoryId
+    );
+
+    struct DecollateralizeParams {
+        uint[] batchIds;
+        uint[] amountsIn;
+        uint[] amountsOutMin;
+    }
+
     function router() external view returns (address);
 
     function weth() external view returns (address);
 
     function swManager() external view returns (address);
 
     function forwardContractBatch() external view returns (address);
+
+    /// @notice Zap function that achieves the following:
+    /// 1. Swaps `inputToken` to `crispToken` via encoded swap
+    /// 2. Resulting tokens are decollateralized to forward credits via SolidWorldManager
+    /// 3. The resulting forward credits are transferred to `msg.sender`
+    /// 4. Any remaining crisp token balance of SolidZapDecollateralize will be transferred to the `dustReceiver`
```

Comment:
nit: In order to be consistent I suggest sticking to either passive or active voice.
1. Swaps
2. Decollaterizes
3. Transfers
4. Transfers

or
1. Tokens are swapped
2. Tokens are decollaterized
3. Credits are transferred
4. Tokens are transferred